torch/csrc/jit/codegen/cuda/scheduler/mma_utils.cpp

```diff
@@ -232,6 +283,42 @@ void validateResultInnerMN(TensorView* tv, int m, int n) {
       tv->getMaybeRFactorDomain()[root_dim - 1], tv->axis(-1), n));
 }
 
+//! Performs checks on tv given to schedule ld matrix.
+//!  Currently only allowed ones are either:
+//!    1. direct output of an ldmatrix op  or
+//!    2. direct output of a broadcast op following a ldmatrix op
+//!  Returns true if the tv is an immediate output of ldmatrix op
+//!
+//! TODO: this check is
```

Comment:
Nit: what's the todo for?

Reply:
It's for eventually removing the pattern matching for ldmatrix. Completed the todo comment. Thanks.